tidb: add tidb_analyze_column_options (#18281)
pingcap/docs: benchmark/benchmark-tidb-using-ch.md

## Collect statistics

> To ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics in advance. **We need to set [tidb_analyze_column_options](/system-variables.md#tidb_analyze_column_options-new-in-v830) to `ALL`, otherwise it will result in a significant drop in query performance.**

Reviewer: Please limit the use of first person "we".
```suggestion
To ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics in advance. **Be sure to set [`tidb_analyze_column_options`](/system-variables.md#tidb_analyze_column_options-new-in-v830) to `ALL`, otherwise collecting statistics can result in a significant drop in query performance.**
```

> To ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics in advance. **Be sure to set [`tidb_analyze_column_options`](/system-variables.md#tidb_analyze_column_options-new-in-v830) to `ALL`, otherwise collecting statistics can result in a significant drop in query performance.**

Reviewer 2: ```suggestion
To ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics in advance. **Be sure to set [`tidb_analyze_column_options`](/system-variables.md#tidb_analyze_column_options-new-in-v830) to `ALL`. Otherwise collecting statistics can result in a significant drop in query performance.**
```
